dhis-2/dhis-api/src/main/java/org/hisp/dhis/reservedvalue/ReservedValueStore.java

```diff
@@ -42,8 +42,6 @@ public interface ReservedValueStore extends GenericStore<ReservedValue> {
   List<ReservedValue> getAvailableValues(
       ReservedValue reservedValue, List<String> values, String ownerObject);
 
-  List<ReservedValue> reserveValuesJpa(ReservedValue reservedValue, List<String> values);
-
```

Comment:
This is a code clean up done by Luca from tracker team. He helped to fix one of the unit test failed after using entityManager.